server/test/controllers/BlocksControllerSpec.scala

```diff
@@ -367,6 +368,7 @@ class BlocksControllerSpec extends MyAPISpec {
       val blockheader =
         block
           .into[BlockHeader.Simple]
+          .withFieldConst(_.transactions, 0)
```

Comment:
It's important to test new endpoints, so that we get failing tests if we ever break the API compatibility